indigo_app/templates/indigo_api/work_form.html

```diff
@@ -80,28 +89,40 @@ <h5 class="card-header">{% trans 'Work identification' %}</h5>
 
           <div class="form-group col-4">
             <label for="{{ form.frbr_actor.id_for_label }}">{% trans 'Responsible organisation (actor)' %}</label>
-            <input type="text" class="form-control" id="{{ form.frbr_actor.id_for_label }}"
-                   name="{{ form.frbr_actor.html_name }}"
-                   value="{{ form.frbr_actor.value|default:'' }}">
-            <div class="form-text text-muted">{% trans 'Organisation that created this work. Not normally used for acts.' %}</div>
+            <input
+                hx-trigger="change, keyup delay:500ms changed"
+                hx-target=".duplicate-wrapper"
+                hx-post="{% url 'work_form_find_duplicates' place.place_code %}"
+                type="text" class="form-control" id="{{ form.frbr_actor.id_for_label }}"
+                name="{{ form.frbr_actor.html_name }}"
+                value="{{ form.frbr_actor.value|default:'' }}">
+            <div class="form-text text-muted">{% trans 'Organisation that created this work. Not normally used for Acts.' %}</div>
           </div>
         </div>
 
         <div class="form-row">
           <div class="form-group col-md-3 mb-0">
             <label for="{{ form.frbr_date.id_for_label }}" class="required">{% trans 'Year (or date) of introduction' %}</label>
-            <input type="text" class="form-control" id="{{ form.frbr_date.id_for_label }}"
-                   required pattern="\d{4}(-\d{2}-\d{2})?" placeholder="yyyy(-mm-dd)"
-                   name="{{ form.frbr_date.html_name }}"
-                   value="{{ form.frbr_date.value|default:'' }}">
+            <input
+                hx-trigger="change, keyup delay:500ms changed"
```

Comment:
shouldn't all of these triggers be the same?

Reply:
some are input and some are select — each type should all be the same. I've only made the title one post on load though, then it just gets done once when the work form loads and each time one of the fields change.